src/components/PageFooter/PageFooter.tsx

```diff
@@ -49,7 +49,7 @@ const FooterContainer = styled('div')`
   @media (min-width: 1114px) {
```

Comment:
Please use/add helper
https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300
https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0